server/models/handlers/authenticate.js

```diff
@@ -20,8 +23,15 @@ async function getUserFromCredentials(email, password) {
 };
 
 // Verify the access token for each private request to improve security.
+// Returns false if the access token does not match a student nor a worker's token.
 async function isAccessTokenValid(accessToken) {
-    return true;
```

Comment:
I see that you removed the hard coded aspect and replaced it with logic, as per your story!